tests/z_api_config_test.c

```diff
@@ -1,49 +0,0 @@
-//
```

Comment:
Could we still have a test verifying that get/set operations work for the config ?

Reply:
We already have it in z_api_alignment_test.c